package.json

```diff
@@ -34,13 +35,21 @@
     "@devtools-ds/object-inspector": "^1.1.2",
     "@devtools-ds/object-parser": "^1.2.1",
     "@devtools-ds/table": "^1.1.2",
-    "@devtools-ds/themes": "^1.1.2",
+    "@devtools-ds/themes": "^1.2.1",
+    "@devtools-ui/plugin": "0.0.2--canary.13.662",
     "@emotion/styled": "^11",
     "@oclif/core": "1.9.0",
     "@oclif/plugin-legacy": "^1.2.7",
     "@oclif/plugin-plugins": "^1.9.0",
-    "@player-ui/player": "0.7.1-next.0",
-    "@player-ui/types": "0.7.1-next.0",
+    "@player-tools/dsl": "^0.5.1",
+    "@player-ui/common-types-plugin": "0.7.1",
+    "@player-ui/expression-plugin": "^0.7.1",
+    "@player-ui/player": "0.7.1",
+    "@player-ui/pubsub-plugin": "0.7.1",
+    "@player-ui/react": "0.7.1",
+    "@player-ui/reference-assets-components": "0.7.1",
+    "@player-ui/reference-assets-plugin-react": "0.7.1",
+    "@player-ui/types": "0.7.1",
```

Comment:
Any reason to downgrade these?

Reply:
Since we have `0.7.1` on the [devtools-assets](https://github.com/player-ui/devtools-assets/blob/b8e14f2cda861422a30865802011b066b042a287/package.json#L35) I need to keep the same version here, otherwise I fall into the multiple Player instances issue. So, since I needed to opt for a version, I selected the "stable", instead of the `next`.

Reply:
`0.7.1` is later than `0.7.1-next.N` so this is an upgrade